index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./output.css">
+    <script src="./script.js" defer></script>
+    <title>Fake OS</title>
+</head>
+<body>
+   <main class="background-image">
+    <div class="parent">
+        <div class="child">
+            <div class="menu">
+                <div class="combined">
+                    <span style="margin-left: 92%; color:rgb(168, 161, 161);font-weight: bold;font-size: medium;" id="menuCloseSpan">x</span>
+                    <!-- the blue sidebar-->
+                    <div class="menu-bar-side"></div>
+                    <!--Menu starts here-->
+                    <div class="iconPlusNames">
+                        <div class="iconWithNames" onclick="myfunc('Programs', 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s')">
+                            <img src="./images/program.png">
+                            <p>Programs</p>
+                        </div>
+                        <div class="iconWithNames poo" onclick="myfunc('Photos', 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s')">
+                            <img src="./images/img icon.png">
+                            <p>Photos</p>
+                        </div>
+                        <div class="iconWithNames" onclick="myfunc('Applications', 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s')">
```

Comment:
I would move this to js files to have separation from html since this project is vanilla js, you can store the values of your textxs in an array